### PR TITLE
added PyQt.Qt

### PR DESCRIPTION
**Pull request by mbernasocchi**

### CODE REVIEW

File: python/PyQt/PyQt4/Qt.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+
+"""
+***************************************************************************
+    QtCore.py
+    ---------------------
+    Date                 : September 2016
+    Copyright            : (C) 2015 by Marco Bernasocchi
+    Email                : matthias at opengis dot ch
```
Comment (m-kuhn) [inline python/PyQt/PyQt4/Qt.py]: It's ok if you put your address here ;)

File: python/PyQt/PyQt4/Qt.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+
+"""
+***************************************************************************
+    QtCore.py
```
Comment (m-kuhn) [inline python/PyQt/PyQt4/Qt.py]: The filename doesn't match

File: python/PyQt/PyQt4/Qt.py (hunk 1 above)
Comment (mbernasocchi) [inline python/PyQt/PyQt4/Qt.py]: for this time yes, I guess Pycharm does not understand _at_ adresses ;)

File: python/PyQt/PyQt4/Qt.py (hunk 2 above)
Comment (mbernasocchi) [inline python/PyQt/PyQt4/Qt.py]: ?

Comment (mbernasocchi) [inline python/PyQt/PyQt4/Qt.py]: ah sorry I saw

**Comment by mbernasocchi** [comment]
@m-kuhn could you put this in master_2 also?

**Comment by m-kuhn** [comment]
Yep, done

**Comment by mbernasocchi** [comment]
great

**Comment by mbernasocchi** [comment]
oh, and 2.14 pretty please

**Comment by mbernasocchi** [comment]
that should do it @m-kuhn. thanks

**Pull request merged**